src/commands/build-util.ts

```diff
@@ -22,28 +27,45 @@ export function isDirectoryImport(fileLoc: string, spec: string): boolean {
   return false;
 }
 
-export function wrapImportMeta(code: string, {hmr, env}: {hmr: boolean; env: boolean}) {
+export function wrapImportMeta({
+  code,
+  hmr,
+  env,
+  buildOptions,
+}: {
+  code: string;
+  hmr: boolean;
+  env: boolean;
+  buildOptions: SnowpackConfig['buildOptions'];
+}) {
   if (!code.includes('import.meta')) {
     return code;
   }
   return (
     (hmr
-      ? `import * as  __SNOWPACK_HMR__ from '/__snowpack__/hmr.js';\nimport.meta.hot = __SNOWPACK_HMR__.createHotContext(import.meta.url);\n`
+      ? `import * as  __SNOWPACK_HMR__ from '/${buildOptions.metadataDir}/hmr.js';\nimport.meta.hot = __SNOWPACK_HMR__.createHotContext(import.meta.url);\n`
       : ``) +
     (env
-      ? `import __SNOWPACK_ENV__ from '/__snowpack__/env.js';\nimport.meta.env = __SNOWPACK_ENV__;\n`
+      ? `import __SNOWPACK_ENV__ from '/${buildOptions.metadataDir}/env.js';\nimport.meta.env = __SNOWPACK_ENV__;\n`
       : ``) +
     '\n' +
     code
   );
 }
 
-export async function wrapCssModuleResponse(
-  url: string,
-  code: string,
-  ext: string,
+export async function wrapCssModuleResponse({
+  url,
+  code,
+  ext,
   hasHmr = false,
-) {
```

Comment:
I personally like converting function params to an option whenever it gets > 4, especially when some are optional. I never can remember the ordering 😅